UI/Panels/Settings/MobiFlightPanel.cs

```diff
@@ -335,7 +335,7 @@ private TreeNode mfModulesTreeView_initNode(MobiFlightModuleInfo moduleInfo, Tre
                     // MultiplexerDrivers should not appear in the tree, therefore they are stored in a dictionary 
                     // (by module name) for easy retrieval
                     if(device.Type == DeviceType.MultiplexerDriver) {
-                        moduleMultiplexerDrivers.Add(moduleInfo.Name, device as MobiFlight.Config.MultiplexerDriver);
+                        moduleMultiplexerDrivers.Add(moduleNode.Text, device as MobiFlight.Config.MultiplexerDriver);
```

Comment:
What is wrong with moduleInfo.Name?
I know both are the same at this point. What was the intention to change it?

Reply:
All other properties here are set using `moduleNode`; exactly because `moduleNode.Text` is the same as `moduleInfo.Name` makes it more logical and less misleading to use it.